File: src/com/zxy/ijplugin/wechat_miniprogram/lang/wxss/WXSSLanguage.java

package com.zxy.ijplugin.wechat_miniprogram.lang.wxss;

import com.intellij.lang.Language;
import com.intellij.openapi.fileTypes.LanguageFileType;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

public class WXSSLanguage extends Language {

    public static WXSSLanguage INSTANCE = new WXSSLanguage();

    private WXSSLanguage() {
        super("WXSS");
    }

    @NotNull
    @Override
    public String getDisplayName() {
        return super.getDisplayName();
    }

    @Nullable
    @Override
    public LanguageFileType getAssociatedFileType() {
        return WXSSFileType.INSTANCE;
    }
}

File: src/main/kotlin/com/zxy/ijplugin/wechat_miniprogram/lang/wxss/WXSSLanguage.kt
package com.zxy.ijplugin.wechat_miniprogram.lang.wxss

import com.intellij.lang.Language
import com.intellij.openapi.fileTypes.LanguageFileType

class WXSSLanguage private constructor() : Language("WXSS") {
    companion object {
        @JvmField
        var INSTANCE = WXSSLanguage()
    }

    override fun getAssociatedFileType(): LanguageFileType? {
        return WXSSFileType.INSTANCE
    }
}
